docs/source-app/examples/github_repo_runner/app.py

```diff
@@ -42,10 +42,10 @@ def __init__(
             script_args=script_args,
             cloud_compute=cloud_compute,
             cloud_build_config=BuildConfig(requirements=requirements),
-            **kwargs,
         )
         self.id = id
         self.github_repo = github_repo
+        self.kwargs = kwargs
```

Comment:
I don't think these are used anywhere.

Reply:
@tchaton ?

Reply:
Yes, this is a bug, should be passed to the init method.